### PR TITLE
Login

### PR DESCRIPTION
**Pull request by wu-reni**

### CODE REVIEW

**Comment by borne-diego** [comment]
We should make sure to update "package.json" so that it includes all required dependencies.

File: Website/web/src/context/AuthContext.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React, { createContext, useState, useContext } from 'react';
+
+const AuthContext = createContext();
+
+export const AuthProvider = ({ children }) => {
+    const [isLoggedIn, setIsLoggedIn] = useState(false);
+    React.useEffect(() => {
+        console.log("AuthProvider - isLoggedIn:", isLoggedIn);
```
Comment (borne-diego) [inline Website/web/src/context/AuthContext.jsx:8]: Left in console log.

**Comment by wu-reni** [comment]
> We should make sure to update "package.json" so that it includes all required dependencies.

**Comment by borne-diego** [comment]
Warning is logged upon user login: "An unhandled error was caught from submitForm() TypeError: setStatus is not a function"

**Pull request merged**